LibreNMS/OS/Arubaos.php

```diff
@@ -217,4 +224,39 @@ public function pollWirelessFrequency(array $sensors)
     {
         return $this->pollWirelessChannelAsFrequency($sensors, [$this, 'decodeChannel']);
     }
+
+    public function discoverFdbTable(): Collection
+    {
+        $fdbt = new Collection;
+
+        $dot1qTpFdbPort = $this->dot1qTpFdbPort();
+
+        $dot1dBasePortIfIndex = SnmpQuery::walk('BRIDGE-MIB::dot1dBasePortIfIndex')->table();
```

Comment:
BridgeMib.php trait includes an an ifIndexFromBridgePort.  That should probably be extended to allow vlan context.